Swap.jl

```diff
@@ -0,0 +1,66 @@
+include("Solution.jl")
+
+DEBUG = true
+
+struct Swap
+    data::TSP
+    solution::Solution
+ end
+ 
+function nodeCost(route::Vector{Int}, index::Int64, weights::Matrix{Float64})
```

Comment:
I think this should be in a separate `Utils.jl` file, it doesn't receive a `Swap` type, and could be useful in future neighborhoods. Also the function is missing the `::Float64` return type.